cmd/olm/main.go

```diff
@@ -100,12 +100,11 @@ func main() {
 	http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusOK)
 	})
-	// TODO: both of the following require vendor updates (add k8s.io/apiserver and update prometheus)
-	//healthz.InstallHandler(mux) //(less code)
-	//mux.Handle("/metrics", promhttp.Handler()) //other form is deprecated
-	http.Handle("/metrics", prometheus.Handler())
 	go http.ListenAndServe(":8080", nil)
 
+	http.Handle("/metrics", promhttp.Handler())
+	go http.ListenAndServe(":8081", nil)
```

Comment:
Also serving a nil handler here

Reply:
Yes, nil is intentional, it should use the "DefaultServeMux".

Reply:
Gotcha - didn't know about DefaultServeMux - I think go didn't have a muxer last time I used `http`? 😂